src/common/server.ts

```diff
@@ -454,14 +462,20 @@ export async function startServer(
           break;
       }
     }),
+    newLSClient.onNotification(ShowMessageNotification.type, (params) => {
```

Comment:
For my understanding. This doesn't apply to `$logTrace`? This is only run when the LSP sends an explicit show message request?

I'm asking because I think that showing a dialog for info and warning traces would be a bit overly aggressive.

Reply:
Yeah, this doesn't apply to `$/logTrace`. This only applies to the https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#window_showMessage request which is what the editor uses to display notifications from the server.